FirstTask.py

```diff
@@ -0,0 +1,201 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, name, password, account_no, balance, account_type):
+        self.name = name
+        self.balance = balance
+        self.account_no = account_no
+        self.account_type = account_type
+        self.password = password
+
+    def account_number(self):
+        return self.account_no
+
+    def pass_code(self):
+        return self.password
+    
+    def check_balance(self):
+        return self.balance
+
+    def check_account_type(self):
+        return self.account_type
+    
+    def deposit_bill(self, bill):
+        if bill >= 0:
+            self.balance += bill
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+    
+class Bank():
+    @staticmethod
+    def login(account_list, account_number, code):
+        sys('clear')
+        check1 = False
+        for i in range(len(account_list)):
+            if account_number == account_list[i].account_number() and code == account_list[i].pass_code():
+                return True,account_list[i]
+        return False
+    
+    @staticmethod
+    def transfer_money(sender_account_number, receiver_account_number, amount):

```

Comment:
The `sender_acc_number` and `receiver_acc_number` should be a string not an object as the name suggests. This method should find both of these account objects in `account_list` and then transfer the money.